caas/kubernetes/provider/bootstrap.go

```diff
@@ -152,22 +154,63 @@ type controllerStacker interface {
 	Deploy() error
 }
 
-func controllerCorelation(broker *kubernetesClient) (string, error) {
-	// ensure controller specific annotations.
-	controllerUUIDKey := k8sutils.AnnotationControllerIsControllerKey(false)
-	_ = broker.addAnnotations(controllerUUIDKey, "true")
+// findControllerNamespace is used for finding a controllers namespace based on
+// it's model name and controller uuid. This function really shouldn't exists
+// and should be looked to removed in 3.0. We have it hear as we are still
+// trying to use Kubernetes annotations as database selectors in some parts of
+// Juju.
+func findControllerNamespace(
+	client kubernetes.Interface,
+	modelName string,
```

Comment:
If we are finding the controller namespace, there's no need to pass in a model name.
The caller of this method already ensures that this method is only ever called with model name equal to `environsbootstrap.ControllerModelName`. So we can drop the model name param and use that const